CHANGELOG.md

```diff
@@ -37,6 +37,9 @@ accidentally triggering the load of a previous DB version.**
 * #5462 Fix segfault after column drop on compressed table
 * #5543 Copy scheduled_jobs list before sorting it
 * #5497 Allow named time_bucket arguments in Cagg definition
+* #5543 Copy scheduled_jobs list before sorting it
+* #5570 Improve interpolate error message on datatype mismatch
+* #5558 Use regrole for job owner
```

Comment:
Just ignore this, I'll overwrite it with the correct changelog from main. This is the consequence of "union merge" that we enabled for this file.